### PR TITLE
fix: doc/source/how-to/releasing.rst: fix variables

### PR DESCRIPTION
**Pull request by anscfrisson**
- f0a6e25 `doc/source/how-to/releasing.rst`: use double quotes for variable expansion (otherwise `INDEX_URL` literally equals `https://$PYANSYS_PYPI_PRIVATE_READ_PAT@pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/simple/`)
- f002664 `doc/source/how-to/releasing.rst`: fix powershell nested variable

### CODE REVIEW

**Review by RobPasMue — APPROVED** [review]
LGTM!

**Comment by anscfrisson** [comment]
@RobPasMue Thanks for having approved this PR.

I have added another commit after you approved: f002664 `doc/source/how-to/releasing.rst`: fix powershell nested variable.

The [pre-commit.ci](https://results.[pre-commit.ci](https://results.pre-commit.ci/run/github/395735321/1713970090.9HvmA6H-Q_GXBtn0w1UuZQ)/run/github/395735321/1713970090.9HvmA6H-Q_GXBtn0w1UuZQ) job over the first 2 commits initially failed with:
```error
timeout waiting for merge ref
```

I tested my changes locally with  `pre-commit run --files doc/source/how-to/releasing.rst` which did not change the file.

Since I could not re-run the precommit.ci job, I tried to bypass the pre-commit.ci checks by pushing a commit: `git commit --allow-empty -m"[skip pre-commit.ci]"`, but the [pre-commit.ci](https://results.[pre-commit.ci](https://results.pre-commit.ci/run/github/395735321/1713970834.FxRExZUdRKS7q7LK8_8w6w)/run/github/395735321/1713970834.FxRExZUdRKS7q7LK8_8w6w) passed anyway.

What are the usual ways to solve pre-commit.ci jobs failing with `timeout waiting for merge ref`?

**Comment by RobPasMue** [comment]
First time I see those fails... pretty weird! Seems like something transient anyway. Thanks for the PR. Merging now!

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**
